frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -47,37 +68,88 @@ const StyledTopRow = styled('div')(({ theme }) => ({
     width: '100%',
 }));
 
+const UseButton = styled(Button)(({ theme }) => ({
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.primary.contrastText,
+    '&:hover': {
+        backgroundColor: theme.palette.primary.dark,
+    },
+}));
+
+const PreviewButton = styled(Button)(({ theme }) => ({
+    backgroundColor: 'transparent',
```

Comment:
Should this actually be transparent? What if you have so much text in the description that it runs to the end of the container and underneath the button. That might make the button text hard to read. Would making it the normal background color be better? Or it might not be a problem 🤷🏼